docs/Documentation/Applications/IDAES_Solvers/ipopt.md

```diff
@@ -0,0 +1,21 @@
+# IPOPT + HSL
```

Comment:
Olivia wrote some documentation for IPOPT and HSL in the Libraries section (https://nrel.github.io/HPC/Documentation/Development/Libraries/). Should that be combined with this or moved?